src/content/blog/drupalcon-barcelona-2024.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "DrupalCon Barcelona Wrap-up"
+pubDate: 2024-09-30
+#modifiedDate: 2024-09-06
+summary: DDEV happenings at DrupalCon Barcelona
+author: Randy Fay
+#featureImage:
+#  src: /img/blog/2024/09/ddev-blog.png
+#  alt: DDEV running in the GitLab Pipeline
+#  credit: "[Milad Fakurian](https://unsplash.com/de/@fakurian) (Background)"
+categories:
+  - Community
+---
+
+I spent the last week in Barcelona, Spain for DrupalCon Barcelona. It was so great to meet old friends and new, and to workk with all to integrate new contributors to Drupal.
+
+## BoFs (Birds-of-a-feature) sessions
```

Comment:
```suggestion
## BoFs (Birds of a Feather) sessions
```

It's Feather, right?
https://www.drupal.org/association/blog/the-drupalcon-experience-birds-of-a-feather-bof-sessions

Reply:
Wow, fingers on autopilot. Thanks.